tests/10apidoc/30room-create.pl

```diff
@@ -313,7 +312,14 @@ sub matrix_create_room
    )->then( sub {
       my ( $body ) = @_;
 
-      Future->done( $body->{room_id}, $body->{room_alias} );
+      # The /createRoom response can contain a non-standard room
+      # alias reply, which use has been deprecated.
+      if (defined $body->{room_alias}) {
+         assert_eq( $body->{room_alias}, $opts{rom_alias_name},
+            "room alias" );
+      }
+
+      Future->done( $body->{room_id}, $opts{room_alias_name} );
```

Comment:
This doesn't look quite right. `room_alias_name` is just the localpart of the alias (see https://matrix.org/docs/spec/client_server/r0.6.1#post-matrix-client-r0-createroom), whereas the `room_alias` returned by synapse (and that expected by the rest of sytest is the *complete* alias, including `#` and the domain part.